src/components/field/field.scss

```diff
@@ -1,3 +1,5 @@
+@import '../encoding-pane/filter-shelf.scss';
```

Comment:
Actually looking at `encoding-pane/filter-shelf.scss` I think we probably won't need this as you use this for timeUnitChanger as we probably should use FunctionPicker anyway. :)